Pull Request 1 with Revisions to #1 and #6

namitasshah opened the pull request.

namitasshah committed:
Added 6.1 image

namitasshah committed:
Updated 06 w/Image 1

namitasshah committed:
Added activity logo

namitasshah committed:
Added activity icon to 06

namitasshah committed:
Added 1.1 image

namitasshah committed:
Added 5 W's to 01

namitasshah committed:
Small edit - gif to 01

namitasshah committed:
Added .svg files to images directory

namitasshah committed:
Updated 01-introduction.md

namitasshah committed:
Updated 02-pair_programming.md

namitasshah committed:
Updated 04-code-reviews.md

namitasshah committed:
Updated 06-code-review-activity.md

namitasshah committed:
Added project life cycle graph

namitasshah committed:
Updated 01-introduction.md w/cycle graph

troycomi commented:
The new figures are looking awesome.  From quickly glancing through, there is a broken link in 06 that links to activity.png (which isn't in the repo).  Is there something else it should point to?

May make some other changes before merging, but nice work!

namitasshah committed:
Fixed broken link in 06-code-review-activity.md

namitasshah commented:
Yess, good catch. I removed the extra image.

troycomi committed:
feat: Revise images and labeling

- Renamed images to indicate what they contain
- Removed images as section headers
- Added back some text that was placed in images
- Revised 1.when to clarify intent
- Revised 6.git_workflow to shorten some text
